specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-04-01-preview/examples/createLinkedWorkspace.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "resourceGroup-1",
+    "workspaceName": "workspace-1",
+    "api-version": "2020-04-01-preview",
+    "parameters": {
```

Comment:
Missing path parameter `linkName` here
This is reported by the Model Validation